great_expectations/rule_based_profiler/data_assistant/data_assistant.py

```diff
@@ -332,15 +352,17 @@ def run_profiler_on_data(
     data_assistant: DataAssistant,
     data_assistant_result: DataAssistantResult,
     profiler: BaseRuleBasedProfiler,
+    variables: Optional[Dict[str, Any]] = None,
+    rules: Optional[Dict[str, Dict[str, Any]]] = None,
```

Comment:
I'd love it if we could represent rules as rules instead of dicts

Reply:
@donaldheppner Again, this is important to say as configurations currently, because it has to be exact as what we have in the `BaseRuleBasedProfiler.run()` method.  The purpose is to accept "overrides" (in this use case).  Thanks.